config/analytics.yml

```diff
@@ -118,6 +118,7 @@ shared:
     - in_progress_steps
     - reviewed_at
     - country
+    - email_address
```

Comment:
if i were being pedantic, i would say alphabetical order but you have been, its what was there before so dont worry...

Reply:
Yeah typically I would but I saw the rest weren't, so just stuck it on at the end!

Reply:
fair enough

Reply:
Is it perhaps following the order in schema.db? If so that makes it easier to compare the two